tensorflow/examples/learn/resnet.py

```diff
@@ -141,13 +135,14 @@ def res_net(x, y, activation=tf.nn.relu):
   return learn.models.logistic_regression(net, y)
 
 # Download and load MNIST data.
-mnist = input_data.read_data_sets('MNIST_data')
+mnist = learn.datasets.load_dataset('mnist')
 
-# Restore model if graph is saved into a folder.
-if os.path.exists('models/resnet/graph.pbtxt'):
-  classifier = learn.TensorFlowEstimator.restore('models/resnet/')
+# Create a new resnet classifier.
+classifier = learn.TensorFlowEstimator(
+    model_fn=res_net, n_classes=10, batch_size=100, steps=100,
+    learning_rate=0.001, continue_training=True)
 
-while True:
+for step in xrange(100):
```

Comment:
fit() should never be called in a loop. I know this is from an old example and you didn't do it, but as part of moving this example to the learn/ directory we have to update it to current best practices.

Instead, use steps=100, and if you want to output a score regularly, use a ValidationMonitor.